aws_lambda_builders/workflows/java_gradle/gradlew_resolver.py

```diff
@@ -0,0 +1,29 @@
+"""
+Gradle executable resolution
+"""
+
+from aws_lambda_builders.path_resolver import PathResolver
+
+
+class GradlewResolver(object):
+    DUMMY_PATH = object()
```

Comment:
Can't you just use None instead?

Reply:
I tried just returning `None` but I get a workflow error when running from here: https://github.com/awslabs/aws-lambda-builders/blob/681d65b76c925980a158fd767cc5716488df0474/aws_lambda_builders/workflow.py#L56-L58